src/clj/rems/db/test_data.clj

```diff
@@ -441,93 +439,57 @@
       (catalogue/create-catalogue-item-localization! {:id id :langcode lang :title title}))
     id))
 
-(defn trim-value-if-longer-than-fields-maxlength [value maxlength]
-  (if (and maxlength (> (count value) maxlength))
-    (subs value 0 maxlength)
-    value))
-
 (defn- run-and-check-dynamic-command! [cmd]
   (let [result (applications/command! cmd)]
     (assert (nil? result) {:actual result})
     result))
 
-(defn- create-draft! [user-id catids wfid field-value & [now]]
-  (let [app-id (legacy/create-new-draft-at-time user-id wfid (or now (time/now)))
-        catids (if (vector? catids) catids [catids])
```

Comment:
minor: There used to be the logic here that allowed passing to create-draft! function a single catid that was not in a vector. It seems that all uses of create-draft! pass a single catid, so the parameters of the function could be changed so that it takes a single catid instead of several catids, which cleans up the call-sites a bit.